frontend/test/resolution/testLoopIndexVars.cpp

```diff
@@ -34,6 +34,12 @@
 
 #include <map>
 
+#define ADVANCE_PRESERVING_SEARCH_PATHS_(ctx__) \
+  do { \
+    ctx__->advanceToNextRevision(false); \
+    setupModuleSearchPaths(ctx__, false, false, {}, {}); \
```

Comment:
Strictly speaking this doesn't preserve them as much as it resets them to empty. A macro or method for this that truly preserves search paths would be nice.

Reply:
I'll go ahead and rename the macro to something like "advance preserving standard modules".